source/operations/external-iam.rst

```diff
@@ -10,14 +10,18 @@ External Identity Management
    :local:
    :depth: 2
 
-MinIO supports offloading identity management onto one of the following supported IDentity Providers (IDP):
+MinIO supports multiple external identity managers through the following supported IDentity Providers (IDP):
```

Comment:
I'd drop `supported` at the end of the line. You already said "MinIO supports" at the beginning.